multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/Kemono.kt

```diff
@@ -185,6 +186,14 @@ open class Kemono(
         return post[0].images.mapIndexed { i, path -> Page(i, imageUrl = baseUrl + path) }
     }
 
+    override fun imageRequest(page: Page): Request {
+        var url: StringBuilder = StringBuilder(page.imageUrl)
+        if (preferences.getBoolean(USE_LOW_RES_IMG, false)) {
+            url.insert(baseUrl.length, "/thumbnail")
+        }
+        return GET(url.toString(), headers)
```

Comment:
```suggestion
        val imageUrl = page.imageUrl!!
        if (!preferences.getBoolean(USE_LOW_RES_IMG, false)) return GET(imageUrl, headers)
        val index = imageUrl.indexOf('/', startIndex = 8) // https://
        val url = buildString { 
            append(imageUrl, 0, index)
            append("/thumbnail")
            append(imageUrl, index, imageUrl.length)
        }
        return GET(url, headers)
```

Use `imageUrl.indexOf('/', startIndex = 8)` since `baseUrl` might change (by preference, etc).

You can use the insert method; early return and more `append` lines are just optimizations.